### PR TITLE
Removed PersistentConnection from ASP.NET Core SignalR

### LINKED ISSUE #9358: Add note to SignalR Comparison doc about PersistentConnection
https://docs.microsoft.com/en-us/aspnet/core/signalr/version-differences?view=aspnetcore-2.1

There's no mention of the fact that PersistentConnection was removed in ASP.NET Core. We should call that out.

### PR DESCRIPTION
**Pull request by bradygaster**
fixes #9358

### CODE REVIEW

File: aspnetcore/signalr/version-differences.md [hunk 1]
```diff
@@ -67,6 +69,10 @@ ASP.NET Core SignalR now supports [streaming data](xref:signalr/streaming) from
 
 The ability to pass arbitrary state between clients and the hub (often called HubState) has been removed, as well as support for progress messages. There is no counterpart of hub proxies at the moment.
 
+### PersistentConnection Deprecation
+
+In ASP.NET Core SignalR, the [`PersistentConnection`](https://docs.microsoft.com/en-us/previous-versions/aspnet/jj918097(v%3dvs.118)) class has been removed.
```
Comment (scottaddie) [inline aspnetcore/signalr/version-differences.md]:
```suggestion
In ASP.NET Core SignalR, the [PersistentConnection](https://docs.microsoft.com/previous-versions/aspnet/jj919047(v%3dvs.118)) class has been removed. 
```

File: aspnetcore/signalr/version-differences.md [hunk 2]
```diff
@@ -67,6 +69,10 @@ ASP.NET Core SignalR now supports [streaming data](xref:signalr/streaming) from
 
 The ability to pass arbitrary state between clients and the hub (often called HubState) has been removed, as well as support for progress messages. There is no counterpart of hub proxies at the moment.
 
+### PersistentConnection Deprecation
```
Comment (scottaddie) [inline aspnetcore/signalr/version-differences.md]:
```suggestion
### PersistentConnection deprecation
```

Comment (tdykstra) [inline aspnetcore/signalr/version-differences.md]: If the class doesn't exist in Core, it's not deprecated, it just doesn't exist and that's a difference between the two, not deprecation.

It's still viable in 4.x SignalR unless it's deprecated there in which case the deprecation comment is not about Core.

Comment (bradygaster) [reply aspnetcore/signalr/version-differences.md]: Makes sense. Changed the terminology - let me know if further changes are needed. Thanks!

File: aspnetcore/signalr/version-differences.md [hunk 3]
```diff
@@ -67,6 +69,10 @@ ASP.NET Core SignalR now supports [streaming data](xref:signalr/streaming) from
 
 The ability to pass arbitrary state between clients and the hub (often called HubState) has been removed, as well as support for progress messages. There is no counterpart of hub proxies at the moment.
 
+### PersistentConnection Removal
```
Comment (tdykstra) [inline aspnetcore/signalr/version-differences.md]: Keep Removal and lowercase it or omit it, your choice. It still exists in 4.x, so I think omitting "Removal" makes sense.  We didn't really "remove" it from Core, we just never added it.
```suggestion
### PersistentConnection
```

**Comment by scottaddie** [comment]
Thank you for another awesome PR, @bradygaster!

**Pull request merged**